src/main/java/com/google/devtools/build/lib/starlarkbuildapi/python/PyRuntimeInfoApi.java

```diff
@@ -204,6 +215,15 @@ interface PyRuntimeInfoProviderApi extends ProviderApi {
                       + "Default is <code>"
                       + DEFAULT_STUB_SHEBANG
                       + "</code>."),
+          @Param(
+              name = "bootstrap_template",
+              // allowedTypes = {
+              //   @ParamType(type = FileApi.class),
+              // },
```

Comment:
This allowed types looks correct, but it's commented out. Did you run into some problem?

Reply:
Yes, if I enable this I get something like "Target is not a FileApi"

Reply:
Ah ok. That sounds like the ctx.attr vs ctx.file comment I have further down -- pass ctx.file.boostrap_template instead and it should go away.